templates/patterns/banners/pattern-banner--variant-hero.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
```

Comment:
Following my comment above, I don't think this variant file is needed (the hero variant doesn't exist anymore)